src/styles/blocks/card-block.scss

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  font-style: $main-font-style;
+  color: $main-text-color;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img-imac {
+    width: 160px;
+    height: 134px;
+  }
```

Comment:
The class name `&__img-imac` should follow the BEM naming convention. Consider renaming it to `.container__img-imac` or similar.